prow/cmd/deck/static/script.js

```diff
@@ -138,6 +138,8 @@ function redraw() {
         r.appendChild(stateCell(build.state));
         if (build.pod_name) {
             r.appendChild(createLinkCell("\u2261", "log?pod=" + build.pod_name));
+        } else if (build.url !== "") {
+            r.appendChild(createLinkCell("G", build.url));
```

Comment:
Whoops, I didn't notice this when I reviewed. This seems wrong, we now have two links to `build.url` which is strange. Plus, the G probably stands for Gubernator, but doesn't make sense if the link doesn't actually go to Gubernator.